spec/support/europeana_api_helper.rb

```diff
@@ -3,11 +3,28 @@ module EuropeanaAPIHelper
   RSpec.configure do |config|
     config.before(:each) do
       # API Search
+      
+      items = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.